slidez-fe-core/src/common/components/forms/sign-form.scss

```diff
@@ -91,14 +101,14 @@
   align-items: center;
   text-align: center;
   font-size: 18px;
-  color: $grey;
+  color: #E5E5E5;
```

Comment:
Why would you replace variable color with just inline color?

Reply:
As this color is used here for one time and I don't like to create such variables